algorithm-simulator/build.gradle

```diff
@@ -34,15 +33,12 @@ tasks.withType(ScalaCompile) {
     scalaCompileOptions.additionalParameters = gradle.scala.compileFlags
 }
 
-mainClassName = 'style95.Simulator'
-
 task runScenario(type: JavaExec) {
     classpath = sourceSets.main.runtimeClasspath
 
-    if (!project.hasProperty("scenario")) {
-        throw new GradleScriptException("runScenario task should a Pscenario argument")
```

Comment:
This exception was thrown when running `./gradlew test`. It seems that gradle runs this section in every. It might has something to do with how `JavaExec` is implemented. As a work around, I'll remove this excetion.